src/backend/src/args.js

```diff
@@ -0,0 +1,35 @@
+import { parseArgs } from 'util'
+
+export default function args() {
+  const { values } = parseArgs({
+    options: {
+      sentryEnvironment: { type: 'string' },
+      sentryUserId: { type: 'string' },
+      metricsIsEnabled: { type: 'boolean', default: false },
+      version: { type: 'string' },
+      rootKey: { type: 'string' },
+    },
+    strict: true,
+  })
+
+  const {
+    sentryEnvironment,
+    sentryUserId,
+    metricsIsEnabled,
+    version,
+    rootKey,
+  } = values
+
+  if (typeof metricsIsEnabled !== 'boolean')
+    throw new Error('backend did not receive metricsIsEnabled')
+  if (typeof sentryUserId !== 'string')
+    throw new Error('backend did not receive sentryUserId')
+  if (typeof sentryEnvironment !== 'string')
+    throw new Error('backend did not receive sentryEnvironment')
+  if (typeof values.rootKey !== 'string')
+    throw new Error('backend did not receive root key from front end')
+  if (typeof values.version === 'string')
```

Comment:
```suggestion
  if (typeof values.version !== 'string')
```

Reply:
This will fix the typecheck that is failing in CI.
```suggestion
  if (typeof version === 'string')
```